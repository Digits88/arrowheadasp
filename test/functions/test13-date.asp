<!-- Test Date function.
-->

DATE = <%=DATE%><br>
