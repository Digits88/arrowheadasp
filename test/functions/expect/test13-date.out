<!-- Test Date function.
-->

DATE = ^[0-9]{1,2}/[0-9]{1,2}/[0-9]{4}$<br>
